parser/ast.go

```diff
@@ -108,6 +108,15 @@ func newServiceElements(semicolons []*ast.RuneNode, elements []ast.ServiceElemen
 	return elems
 }
 
+func newMethodElements(semicolons []*ast.RuneNode, elements []ast.RPCElement) []ast.RPCElement {
```

Comment:
Looks like the first two commits from this branch are unintentional? They are shared with #212.